Sources/Backup/LocalBackup/OASettingsHelper.mm

```diff
@@ -393,7 +393,10 @@ - (void) exportSettings:(NSString *)fileDir fileName:(NSString *)fileName settin
     << OsmAnd::ResourcesManager::ResourceType::SlopeRegion
     << OsmAnd::ResourcesManager::ResourceType::WikiMapRegion
     << OsmAnd::ResourcesManager::ResourceType::DepthMapRegion;
-    NSArray<NSString *> *localIndexFiles = [OAResourcesUIHelper getInstalledResourcePathsByTypes:types];
+    NSMutableArray<NSString *> *localIndexFiles = [[OAResourcesUIHelper getInstalledResourcePathsByTypes:types] mutableCopy];
```

Comment:
It is better to add parameter `includeHidden` to `getInstalledResourcePathsByTypes`